pkgs/development/python-modules/accuweather/default.nix

```
@@ -0,0 +1,30 @@
+{ aiohttp, buildPythonPackage, fetchFromGitHub, lib, pytest, pytestcov
+, pytest-asyncio, python, pytestrunner }:
+
+buildPythonPackage rec {
+  pname = "accuweather";
+  version = "0.0.11";
+
+  src = fetchFromGitHub {
+    owner = "bieniu";
+    repo = pname;
+    rev = version;
+    sha256 = "1sgbw9yldf81phwx6pbvqg9sp767whxymyj0ca9pwx1r6ipr080h";
+  };
+
+  buildInputs = [ pytestrunner ];
+  propagatedBuildInputs = [ aiohttp ];
+
+  checkInputs = [ pytest pytestcov pytest-asyncio ];
+  checkPhase = ''
+    ${python.interpreter} setup.py test
+  '';
```

If you're going to use pytest, we have a helper for that
```suggestion
  nativeBuildInputs = [ pytestrunner ];
  propagatedBuildInputs = [ aiohttp ];

  checkInputs = [ pytestCheckHook pytestcov pytest-asyncio ];
```